sdk/anomalydetector/azure-ai-anomalydetector/src/main/java/com/azure/ai/anomalydetector/AnomalyDetectorClient.java

```diff
@@ -928,8 +928,9 @@ public AnomalyDetectionModel trainMultivariateModel(ModelInfo modelInfo) {
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return response of listing models as paginated response with {@link PagedIterable}.
      */
+    @Generated
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    PagedIterable<AnomalyDetectionModel> listMultivariateModels(Integer skip, Integer top) {
+    public PagedIterable<AnomalyDetectionModel> listMultivariateModels(Integer skip, Integer top) {
```

Comment:
Where to do the configure?

MVAD should have a "partial-update: true"

Reply:
Also how much does it depends on the config on spec repo.

And does it compatible with current design (that package.json does not contain cadl-java)?

Reply:
cadl-location.yaml:
This is used to config cadl spec location and the commit on the spec repo to pull. It does not contain any cadl-project related configuration.

Cadl-Project-Sync.ps1:
This script will pull spec repo files to local folder `TempCadlFiles`.
![image](https://user-images.githubusercontent.com/87355844/216869644-f339124d-9e83-43f3-9240-97ef60bb1e82.png)

Cadl-Project-Generate.ps1:
This script will generate sdk based on the configuration in `TempCadlFiles`.

From my understanding, if we want to generate Java sdk with customized settings, we need to modify the [cadl-project.yaml](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/AnomalyDetector/cadl-project.yaml) from spec repo. Can double check with Micheal.